spec/models/manageiq/providers/embedded_ansible/automation_manager/event_catcher/stream_spec.rb

```diff
@@ -1,4 +1,13 @@
 describe ManageIQ::Providers::EmbeddedAnsible::AutomationManager::EventCatcher::Stream do
+  before do
+    @old_library = VCR.configuration.cassette_library_dir
+    VCR.configure { |c| c.cassette_library_dir = ManageIQ::Providers::AnsibleTower::Engine.root.join('spec/vcr_cassettes') }
+  end
+
+  after do
+    VCR.configure { |c| c.cassette_library_dir = @old_library }
+  end
```

Comment:
I think you can pass the cassette_dir as part of the use_cassette call, and then you don't need the before/after dance.